Error with ROOT::Eve when building DD4hep

Hello!  have an error whem building DD4hep and I think it is connected to using Eve. Nearly a week ago I had no such problem when building DD4hep. ROOT was coorectly built from source as it is stated in the instruction: https://root.cern/install/build_from_source/
Here how I build DD4hep:
```
git clone https://github.com/AIDASoft/DD4hep.git
cd DD4hep
mkdir build && cd build
cmake -DDD4HEP_USE_GEANT4=ON -DBoost_NO_BOOST_CMAKE=ON -DDD4HEP_USE_LCIO=ON -DBUILD_TESTING=ON -DDD4HEP_BUILD_EXAMPLES=ON -DCMAKE_BUILD_TYPE=Release ..
make -j20
make install
cd ..
source bin/thisdd4hep.sh
```
The error happens at the 4th step:
```
CMake Error at DDEve/CMakeLists.txt:21 (target_link_libraries):
  The link interface of target "DDEve_Interface" contains:

    ROOT::Eve

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at UtilityApps/CMakeLists.txt:79 (target_link_libraries):
  Target "teveLCIO" links to:

    ROOT::Eve

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at UtilityApps/CMakeLists.txt:85 (target_link_libraries):
  Target "teveDisplay" links to:

    ROOT::Eve

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
 ```
Here is log from this command: [out.txt](https://github.com/user-attachments/files/19316209/out.txt)

I am using Ubuntu 24.04.2 LTS (GNU/Linux 6.8.0-53-generic x86_64), ROOT version: 6.32.10

> Or that you udpated your local copy of DD4hep to include these changes and you now see them applied in your local copy?
Yes. Also, the amount of errors became less compared to what I have stated before. Now it is only
```
CMake Error at DDEve/CMakeLists.txt:21 (target_link_libraries):
  The link interface of target "DDEve_Interface" contains:

    ROOT::Eve

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
```

--- a/DDEve/CMakeLists.txt
+++ b/DDEve/CMakeLists.txt
@@ -18,6 +18,11 @@ target_include_directories(DDEve_Interface
   $<INSTALL_INTERFACE:include>
 )
 
+if(NOT TARGET ROOT::Eve)
+  MESSAGE(STATUS "ROOT::Eve not found: not building DDEve")
+  return()
+endif()
+
 target_link_libraries(DDEve_Interface INTERFACE ROOT::Core ROOT::Rint ROOT::Eve ROOT::Gui ROOT::Graf3d ROOT::RGL ROOT::EG)
 if(ROOT_VERSION VERSION_GREATER_EQUAL 6.27.00)
   target_link_libraries(DDEve_Interface INTERFACE ROOT::ROOTGeomViewer)

